### DISCUSSION TITLE
Tranparent issue in Avalonia Window

### DISCUSSION THREAD

**Discussion by HossamSalah00** [AvaloniaUI/Avalonia#14444] [General]
![Screenshot 2024-02-01 181858](https://github.com/AvaloniaUI/Avalonia/assets/85931505/37c7245b-e880-4c03-8064-265601592a7c)
I have issue in any window i create and it becomes completely transparent and added user control doesn`t appear .
this problem happened when updating via nuget to avalonia 11.0.7 .

**Comment by maxkatz6** [comment]
Check output logs, or you can add `.LogToTrace()` to your AppBulder in BuildAvaloniaApp() method.
